### DISCUSSION TITLE
Connection refused

### DISCUSSION THREAD

**Discussion by Falci** [supabase/supabase#20318] [Questions]
Hi all,

I just opened an old project, I was using supabase.com for development (now I know how to run everything locally using docker). But I'm about to launch this project, so I was just reviving and checking if everything is ok.
The project on supabase.com was put on hold, which is fine. I re enabled it and updated the password.

But I can't connect to the database. Connection refused. Is this because the project is old and I just restored it?
```
SequelizeConnectionRefusedError: connect ECONNREFUSED 54.153.62.221:5432
```

**Comment by awalias** [comment]
hard to say without the project details, consider opening a ticket if it's something you're planning to continue working on
